src/Dfe.ContentSupport.Web/Views/Shared/RichText/_Entry.cshtml

```diff
@@ -8,6 +8,13 @@
         <div id=@target.JumpIdentifier></div>
     }
 
+    var contentTypeId = Model.Data.Target.Sys.ContentType?.Sys.Id;
```

Comment:
Very minor but probably best use the target variable that is already available? Will make it easier to read

Reply:
[7c18842](https://github.com/DFE-Digital/sts-content-support/pull/88/commits/7c188428a4aa9e7c7039854ba83d5ecde67597c3)